riak_test/yz_solrq_test.erl

```diff
@@ -448,48 +402,51 @@ first_purged([_H|T] = _Written, Searched) ->
 %%
 do_purge([Node|_] = Cluster, PBConn,
          {Bucket1, Index1},
```

Comment:
Since we're removing `Bucket2` and `Index2`, could we remove the 1s that are leftover in all the variable names in this function?